src/software/simulated_tests/simulated_test_fixture.h

```diff
@@ -39,6 +40,13 @@ class SimulatedTestFixture : public ::testing::Test
      */
     void enableVisualizer();
 
+    /**
+     * Creates a directory to output logs to in the directory at the
+     * TEST_UNDECLARED_OUTPUTS_DIR Bazel environment variable, and sets up some
```

Comment:
it's worth linking https://docs.bazel.build/versions/master/test-encyclopedia.html#initial-conditions here

Reply:
added